adoc/architecture-description.adoc

```diff
@@ -380,6 +380,9 @@ Certificates are stored under `/etc/kubernetes/pki` on the control plane nodes.
 |etcd/peer.key                |RSA      |2048
 |===
 
+Please refer to the _{productname} Admin Guide_ for more information on
```

Comment:
You can replace this with: 

```
Please refer to the link:{docurl}single-html/caasp-admin/#_certificates[{productname} Administration Guide] for more information on the rotation of certificates.
```

This automatically rewrites the `{docurl}` attribute to the correct location for the document and links to the certs section.

Reply:
I took this from the another guide. I am fine with changing it, but shouldn't it be changed everywhere else, in a different PR?

Reply:
Either way is fine. You can just fix this PR and tell me where you found the other place and I'll fix it there. If you already know and it's only one or two places feel free to add commits fixing this to this PR here.